src/components/Person/Person.jsx

```diff
@@ -1 +1,41 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age && (
+    <p className="Person__age">
+      {`I am ${person.age}`}
+    </p>
+    )}
+
+    {!person.isMarried
+      ? (<p className="Person__partner">I am not married</p>)
+      : (
+        <p className="Person__partner">
+          {`${person.partnerName} is my ${person.sex === 'f' ? 'husband' : 'wife'}`}
+        </p>
```

Comment:
you can simplify this code

Its better to check inside `p` tag, just create a varible for `partner`  like

`const partner = sex === 'm' ? 'wife' : 'husband';`

so in the end you are gonna have 
```
<p className="Person__partner">
   {isMarried
      ? `My ${partner}'s name is ${partnerName}`
      : 'I am not married'
    }
</p>
 ```